bricks-maven-plugin/src/test/java/com/github/ingogriebsch/bricks/maven/plugin/analyzer/AbstractMavenAnalyzerTest.java

```diff
@@ -19,37 +19,45 @@
  */
 package com.github.ingogriebsch.bricks.maven.plugin.analyzer;
 
-import static org.junit.jupiter.api.Assertions.*;
-import static org.mockito.Mockito.*;
+import static com.github.ingogriebsch.bricks.maven.plugin.analyzer.AnalysisResult.OK;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertSame;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentCaptor.forClass;
+import static org.mockito.Mockito.RETURNS_MOCKS;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
 import com.github.ingogriebsch.bricks.model.Component;
 
-import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 import org.mockito.ArgumentCaptor;
 
 public class AbstractMavenAnalyzerTest {
 
     @Test
     public void testNullContext() throws Exception {
-        Assertions.assertThrows(NullPointerException.class, () -> {
-            NullAnalyzer uut = new NullAnalyzer();
+        NullAnalyzer uut = new NullAnalyzer();
+
```

Comment:
so now, you're into empty lines ? :D yes, i am teasing :D

Reply:
you're splitting arrange/act/assert with newlines? remind me of that when were talking. not saying it is bad, just want to learn about the decisions.

Reply:
1. I'm moving the creation of the analyzer out of the assert because I want to understand what has to be asserted (that augment is throwing and not the ctor in this case).
2. I find it a legal way to structure given/then/when per newlines to understand it easier, yes! Again, I have nothing against newlines in general, I personally do not like to blow up the code because of a newline after every method declaration or before each closing statement.

Reply:
> I'm moving the creation of the analyzer

 sure thing. as i said, this was a programmatic refactoring that tried to change as little as possible and wrapping the whole method body is semantically the same than before (@Test(expected...))

> a newline after every method declaration or before each closing statement.

fair enough